copulas/multivariate/tree.py

```diff
@@ -414,19 +373,57 @@ def is_adjacent(self, another_edge):
         return (self.L == another_edge.L or self.L == another_edge.R or
                 self.R == another_edge.L or self.R == another_edge.R)
 
-    def get_likelihood(self, U):
+    @staticmethod
+    def sort_edge(edge1, edge2):
```

Comment:
Considering that you would always use two instances of `Edge` to call `sort_edge`, and also that the input order should no matter, as we are trying to order them, wouldn't it make more sense to make this `staticmethod`  a regular instance method, consider `edge1` to be `self` and call it from one of the instances to be sorted?

```python
first, second = edge1.sort(edge2)
first, second = Edge.sort_edge(edge1, edge2)
```

Reply:
I changed the arguments such that it could be passed in a list of edges. Thought this would be more convenient.